recipe/meta.yaml

```diff
@@ -70,6 +71,7 @@ outputs:
         - test_server_imports.py
       commands:
         - python test_server_imports.py
+        - python -c "import alchemiscale; assert(alchemiscale.__version__ == '{{ version }}')"
```

Comment:
I honestly didn't know we could use `assert` as a function call. :exploding_head:

Reply:
I think since it is a statement, you can whack `( )` around it, like `return(2 + 3)` works as well as `return 2 + 3`